pkg/csource/csource.go

```diff
@@ -59,9 +59,12 @@ func Write(p *prog.Prog, opts Options) ([]byte, error) {
 		handled[c.Meta.CallName] = c.Meta.NR
 	}
 	for name, nr := range handled {
-		fmt.Fprintf(w, "#ifndef __NR_%v\n", name)
-		fmt.Fprintf(w, "#define __NR_%v %v\n", name, nr)
-		fmt.Fprintf(w, "#endif\n")
+		// Only generate defines for new (added after commit 8a1ab3155c2ac on 2012-10-04) native syscalls.
+		if nr >= 313 && nr <= 10000 {
```

Comment:
Why 10000? I can imagine with have more real syscalls in future. Fake syscalls start with 1000000, but again hardcoding 1000000 will be duplication of code. Use `!strings.HasPrefix(name, "syz_") && nr >= 313". We already use HasPrefix in few places.
Also add a TODO that this assumes linux/amd64 and add a note to #191 that this another place to handle.